src/components/UploaderPanel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from 'react';
+import { Label, ErrorMessage } from 'lbh-frontend-react';
+import styles from '../styles/UploaderPanel.module.scss';
+
+const classNameFromProps = (props: Props) => {
+  let className = `${styles.panel}`;
+  if (props.set) className += ` ${styles.setPanel}`;
+  if (props.error) className += ` ${styles.errorPanel}`;
+  return className;
+};
+
+const UploaderPanel: FunctionComponent<Props> = (props: Props) => (
+  <div className={classNameFromProps(props)}>
+    <Label labelFor={props.name}>{props.label}</Label>
+    {props.hint && (
+      <p className="lbh-body-s" id={`${props.name}-hint`}>
+        {props.hint}
+      </p>
+    )}
+    {props.error && <ErrorMessage>{props.error}</ErrorMessage>}
+    <input
+      type="file"
+      className="govuk-file-upload  lbh-file-upload"
+      name={props.name}
+      id={props.name}
+      data-testid="fileInput"
+      accept="image/*,application/pdf"
+      onChange={(e) => {
+        if (e.currentTarget.files !== null)
+          props.setFieldValue(props.name, e.currentTarget.files[0]);
+      }}
+      aria-describedby={props.hint && `${props.name}-hint`}
+    />
+  </div>
+);
+
+interface Props {
+  setFieldValue(): void;
```

Comment:
I'm getting errors in the tests because this prop type doesn't accept any arguments, but you're giving it 2 arguments in the on change handler. Try changing it to:

```suggestion
  setFieldValue(name: string, value: string): void;
```